integrations/angular2.md

```diff
@@ -107,6 +107,10 @@ export class SimpleTinyComponent implements AfterViewInit, OnDestroy {
       skin_url: 'assets/skins/lightgray',
       setup: editor => {
         this.editor = editor;
+        editor.on('Change', () => {
```

Comment:
You can instead just chain the event names, so just att `change` after `keyup` so it becomes `editor.on('keyup change', () => ...`.